traitsui/wx/ui_panel.py

```diff
@@ -57,7 +57,6 @@
 from .constants import screen_dx, screen_dy, WindowColor
 
 from .ui_base import BaseDialog
-from .constants import is_mac
```

Comment:
This was unused – if it was left there intentionally let me know and I will revert.